### DISCUSSION TITLE
ABORT: SDL: Can't init SDL wayland not available

### DISCUSSION THREAD

**Discussion by HubKing** [dosbox-staging/dosbox-staging#3596] [Q&A] [answered]
When trying to run it with ` export SDL_VIDEODRIVER=wayland`, it crashes with

```
Stack trace:
3             0x4d54de _start + 46
2       0x716c93ad5d8a __libc_start_main + 138
1       0x716c93ad5cd0 /usr/lib/libc.so.6(+0x25cd0) [0x716c93ad5cd0]
0             0x77bb91 sdl_main(int, char**) + 7649
2024-04-12 18:38:43.661 | ABORT: SDL: Can't init SDL wayland not available
Aborted (core dumped)
```

I am using Arch Linux KDE Plasma 6 in Wayland mode. Is there anything I need to install to solve this problem?

**Comment by rderooy** [comment, marked as answer]
It seems your SDL2 library does not have wayland support enabled. You should install or compile a version of SDL2 with wayland support, if you want to avoid using XWayland.
